ts/features/fci/components/DocumentViewer.tsx

```diff
@@ -81,35 +82,32 @@ const renderFooter = (url: string, filePath: string) => {
     accessibilityLabel: I18n.t("messagePDFPreview.save")
   };
 
-  const openButtonProps: ButtonSolidProps = {
-    onPress: () => {
-      ReactNativeBlobUtil.android
-        .actionViewIntent(
-          FciDownloadPreviewDirectoryPath + "/" + getFileNameFromUrl(url),
-          "application/pdf"
-        )
-        .catch(_ => {
-          IOToast.error(I18n.t("messagePDFPreview.errors.opening"));
-        });
-    },
-    label: I18n.t("messagePDFPreview.open"),
-    accessibilityLabel: I18n.t("messagePDFPreview.open")
-  };
+  // const openButtonProps: ButtonSolidProps = {
```

Comment:
Could you clarify why this code is currently commented?

Reply:
The removal of the third button was a suggestion of sorts, so I kept the third button props for quick revert in case of need. But in light of this morning's discussion, I'm removing it.

Reply:
Now we're on the same page! 💫

Reply:
@LeleDallas Removed here → [`a231809` (#6477)](https://github.com/pagopa/io-app/pull/6477/commits/a2318093a02e20db1f36af0226aaa9962f4b2afd)